api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/ResumoMensalCohortQueries.java

```diff
@@ -576,7 +576,7 @@ public CohortDefinition getPatientsWhoHadTbDiagnosticAfterPreArt() {
             + "                          AND e.voided = 0 "
             + "                          AND o.voided = 0 "
             + "                          AND e.encounter_type = ${6} "
-            + "                          AND e.encounter_datetime < DATE_SUB(:endDate, INTERVAL 1 MONTH) "
+            + "                          AND e.encounter_datetime <= DATE_SUB(:endDate, INTERVAL 1 MONTH) "
```

Comment:
@TinoMuianga Please add javadocs for this method

Reply:
Done